### ISSUE TITLE
Request CIRCO validation set title

### ISSUE THREAD

**Issue by ni623** [yzy-bupt/LDRE#6]
Hello, could you please provide the edited CIRCO validation set titles annotated with GPT? The results generated by my own model are not very good, and I would be very grateful

**Comment by sotayang** [comment]
Do you mean the test set? Zero-shot CIR tasks typically don't use training or validation sets for training—they're directly applied to the test set without training. The GPT-generated results for CIRCO and CIRR test sets can be found at:

**CIRCO**: https://github.com/yzy-bupt/LDRE/blob/main/CIRCO/annotations/test.json
**CIRR**: https://github.com/yzy-bupt/LDRE/blob/main/CIRR/cirr/captions/cap.rc2.test1.json
